setup.cfg

```diff
@@ -39,6 +39,7 @@ install_requires =
     psutil>=5.7.2
     poppy>=0.9.0
     pyparsing>=2.2
+    pysiaf>=0.10.0
```

Comment:
So this means that everyone installing the jwst package will now also require installation of pysiaf?  I realize the necessity for doing these level-1b calculations, but it's not needed for any "regular" user-based reprocessing of data products.

Reply:
Good point. Will remove from the general requirements and place into the sdp-only requirements. Also, will see if I can isolate the imports in the code a bit.

Reply:
✅